bolts/data/datasets/audio/ecoacoustics.py

```diff
@@ -128,14 +170,40 @@ def _download_files(self) -> None:
         self.root.mkdir(parents=True, exist_ok=True)
         self.base_dir.mkdir(parents=True, exist_ok=True)
 
-        urls = [
-            "https://zenodo.org/record/1255218/files/AvianID_AcousticIndices.zip",
-            "https://zenodo.org/record/1255218/files/EC_BIRD.zip",
-            "https://zenodo.org/record/1255218/files/UK_BIRD.zip",
-        ]
+        for finfo in self._INDICES_URL_MD5_LIST:
+            if not self._check_integrity(finfo.filename, md5=finfo.md5):
+                download_and_extract_archive(
+                    url=finfo.url, download_root=str(self.base_dir), md5=finfo.md5
+                )
+
+        for finfo in self._URL_MD5_LIST:
+            if not self._check_integrity(finfo.filename, md5=finfo.md5):
+                self.download_and_extract_archive_jar(finfo)
+
+    def download_and_extract_archive_jar(
+        self, finfo: ZenodoInfo, *, remove_finished: bool = False
+    ) -> None:
+        download_url(finfo.url, str(self.base_dir), finfo.filename, finfo.md5)
+
+        archive = self.base_dir / finfo.filename
+        self.log(f"Extracting {archive}")
 
-        for url in urls:
-            download_and_extract_archive(url, str(self.base_dir))
+        suffix, archive_type, _ = _detect_file_type(str(archive))
+        if not archive_type:
+            _ = _decompress(
+                str(archive),
+                str((self.base_dir / finfo.filename).with_suffix("")),
+                remove_finished=remove_finished,
+            )
+            return
+
+        try:
+            subprocess.run(f"jar -xvf {archive}", check=True, cwd=self.base_dir)
```

Comment:
```suggestion
            subprocess.run(["jar", "-xvf", str(archive)], check=True, cwd=self.base_dir)
```

Reply:
if you don't have `shell=True` you need to do the splitting on spaces manually (normally the shell does that for you).